nixos/modules/services/web-apps/limesurvey.nix

```diff
@@ -154,18 +154,21 @@ in
     };
 
     virtualHost = mkOption {
-      type = types.submodule (import ../web-servers/apache-httpd/vhost-options.nix);
+      type = types.submodule (
+        recursiveUpdate
+          (import ../web-servers/nginx/vhost-options.nix { inherit config lib; })
+          { }
+      );
```

Comment:
These `cfg.virtualHost` is not merged with `services.nginx.virtualHosts.${cfg.virtualHost.serverName}` as far as I can see.